GDJS/Runtime/pixi-renderers/runtimescene-pixi-renderer.ts

```diff
@@ -99,6 +112,18 @@ namespace gdjs {
       if (displayAABB) {
         for (let i = 0; i < instances.length; i++) {
           const object = instances[i];
+          const layer = this._runtimeScene.getLayer(object.getLayer());
+          const compensationCameraMovementX =
```

Comment:
That's not really a compensation for "movement" (movement implies a speed), but rather just `cameraTopLeftX` and `cameraTopLeftY` :)